cocos/3d/CCTerrain.cpp

```diff
@@ -159,11 +159,9 @@ void Terrain::onDraw(const Mat4 &transform, uint32_t flags)
 
     if(_isCameraViewChanged )
     {
-        auto camPos = camera->getPosition3D();
-        auto camModelMat = camera->getNodeToWorldTransform();
-        camModelMat.transformPoint(&camPos);
+        auto m = camera->getWorldToNodeTransform();
```

Comment:
Why use `WorldToNodeTransform()` instead of `NodeToWorldTransform()` to get world position for Camera?